### ISSUE TITLE
Additional masses should be added to Stretch and Bounce Screens

### ISSUE THREAD

**Issue by Denz1994** [phetsims/masses-and-springs-basics#5]
During the design meeting on 10/05/18 @kathy-phet and @arouinfar, decided that additional masses should be added to the Stretch and Bounce Screens. On slack @arouinfar mentioned that some mock-ups will be provided.

I have gone ahead and added some preliminary logic for supporting additional masses and shelves based on memory from the meeting. I'm opening this issue for a design review of the suggested changes, once work is complete.

**Comment by Denz1994** [comment]
Adding dev version [1.0.0-dev.2](https://phet-dev.colorado.edu/html/masses-and-springs-basics/1.0.0-dev.2/phet/masses-and-springs-basics_en_phet.html) here for review.

**Comment by arouinfar** [comment]
That was fast @Denz1994! The shelves are looking really good. I'd like to increase the space between the mystery shelf and the springNode, because it's a bit tight currently.

![image](https://user-images.githubusercontent.com/8419308/46564547-30583600-c8c5-11e8-8ff0-7361dfe3c1e5.png)

Here are a few suggestions:
- [ ] Decrease space between these grouped masses by another px or two. The amount of overlap looks good, but I'd like to see the space between each set reduced a bit.
![image](https://user-images.githubusercontent.com/8419308/46564491-e96a4080-c8c4-11e8-9fe8-97d5eaca2515.png)
- [ ] Reduce the length of the shelf of known masses by a few px to accommodate for the above request
- [ ] Move mystery shelf closer to known shelf. Let's try 10 px to start.
- [ ] Allow shelves to float outside of dev bounds, similar to these controls:
     <img src="https://user-images.githubusercontent.com/8419308/46564580-5087f500-c8c5-11e8-80a2-e312c62ed32f.png" width=200>

**Comment by Denz1994** [comment]
The most complex portion of the above suggestion is getting the masses to float along with the shelf because it involves handling a new endAnimationPosition for the "return to shelf" animation when the visible bounds change. I would rather open a new issue for floating masses, as it is quite a logical deviation from MAS.

I just want to make sure this is still needed after the above other changes are made before any further refactors. This also allows @arouinfar to review the already implemented changes.

**Comment by arouinfar** [comment]
@Denz1994 I hadn't taken the "return to shelf" animation into account. While I'm sure QA would have fun with the floating shelf, I'm content to leave the shelf be.

In master, the shelves appear to have shifted, but the masses haven't moved. Compare master (top) to dev.2 (bottom). It looks like you need to re-center the mystery masses, and reduce the spacing between the known masses so they fit on their shelf.

![image](https://user-images.githubusercontent.com/8419308/46590883-08f39b80-ca74-11e8-9549-248e9a9744f1.png)

**Comment by arouinfar** [comment]
I pulled master again, and the shelf alignment was correct. dev.3 also looks good. Thanks @Denz1994.
